extensions/notebook/package.json

```diff
@@ -480,21 +480,23 @@
   },
   "dependencies": {
     "@jupyterlab/services": "^3.2.1",
-    "decompress": "^4.2.0",
+    "@types/adm-zip": "^0.4.32",
+    "@types/tar": "^4.0.3",
+    "adm-zip": "^0.4.14",
```

Comment:
If we're going to use two separate libraries might as stay consistent with our other stuff and use yauzl. (this is also used by vs code in other places)

Reply:
yauzl doesn't support tar AFAIK. Am I wrong?